gatsby-ssr.js

```diff
@@ -1,4 +1,16 @@
+import React from 'react'
+const language_config = require(`./i18n-config.js`)
 import { WrapPagesWithLocaleContext } from './src/components/localization'
 import './src/components/localization/config'
 
 export const wrapPageElement = WrapPagesWithLocaleContext
+
+export function onRenderBody({ setHeadComponents }) {
+    const languageLinks = []
+
+    Object.keys(language_config).map(lang => {
+        languageLinks.push(<link rel="alternate" href="http://deriv.com/about" hrefLang={lang} />)
```

Comment:
pls search for a lifecycle to get the current page, and it should be https 🙏

Reply:
Amam it's not complete :D I had to switch to another task. :D